slides/week3/git.org

```diff
@@ -43,110 +60,84 @@
 
 ** Creating Branches
 - Let's start by creating multiple branches
-
-  #+BEGIN_SRC
-    844a162 * master stable beta Initial commit
-  #+END_SRC
-
-- We have one commit and 3 branches point to the same commit
-  - master, stable, beta
-- Right now, everything is at the same state
 - Create branches with
-
   #+BEGIN_SRC sh
     # git branch <BRANCH_NAME>
 
     # Create two branches, starting from the current commit
 
-    git branch stable
-    git branch beta
+    git branch branch1
+    git branch branch2
   #+END_SRC
 
 ** Switching Branches
 
-- Only one branch can be checked out at a time
+- Only one branch can be worked on at a time
 - ~git status~ will display info on your current branch
 - ~git checkout~ lets you switch between branches
-- Let's checkout the *stable* branch now
+- Let's checkout the ~branch1~ branch now
   #+BEGIN_SRC sh
     git status
     #> On branch master
     #> nothing to commit, working tree clean
 
     # git checkout <BRANCH_NAME>
 
-    git checkout stable
-    #> Switched to branch 'stable
+    git checkout branch1
+    #> Switched to branch 'branch1'
 
     git status
-    #> On branch stable
+    #> On branch branch1
     #> nothing to commit, working tree clean
   #+END_SRC
 
-** First Stable Commit
-- Let's add a commit to the stable branch
+** First branch1 Commit
+#+BEGIN_NOTES
+Do something visual, e.x., draw on whiteboard, gitk
+#+END_NOTES
+- Let's add a commit to the ~branch1~ branch
   #+BEGIN_SRC sh
     echo "Stability counts!" > stable_release.txt
+    git status
     git add -A
+    git status
     git commit -m "Add stable_release.txt"
+    git status
   #+END_SRC
 - This will *not affect* any other branches
 - The other branches will stay behind on 'initial commit'
-  #+BEGIN_SRC
-    5592388 * stable      Add stable_release.txt
-    844a162 * master beta Initial commit
-  #+END_SRC
 
-** First Beta Commit
-- Let's make a experimental commit on the beta branch
+** First branch2 Commit
+- Let's make an experimental commit on the ~branch2~ branch
   #+BEGIN_SRC sh
-    git checkout beta
+    git checkout branch2
     echo "This feature is unstable!" > beta_release.txt
     git add -A
     git commit -m "Add beta_release.txt"
   #+END_SRC
-- Since the beta branch was *behind* the stable branch, this creates a 'fork' in the commit history
-- Notice, master has not moved at all (since no commits have been made on it)
-  #+BEGIN_SRC
-    1c65f7b * beta Add beta_release.txt
```

Comment:
Any reason why the graphs were removed? Trying to hide git's graph structure leads to MUCH more confusion down the road (most people try to hide the graph structure and treat it as a line, which is not the case).

Reply:
the text based renderings were confusing. I think adding speaker notes to pull up something like gitk would be better to show the structure instead of showing them text renderings that do not make any sense if you do not already know what a commit is (which they really do not).